test/nimble.phx.gen.template/addons/readme_test.exs

```diff
@@ -0,0 +1,55 @@
+defmodule Nimble.Phx.Gen.Template.Addons.ReadmeTest do
+  use Nimble.Phx.Gen.Template.AddonCase
+
+  describe "#apply/2" do
+    test "copies the README.md", %{
+      project: project,
+      test_project_path: test_project_path
+    } do
+      in_test_project(test_project_path, fn ->
+        Addons.Readme.apply(project)
+
+        assert_file("README.md", fn file ->
+          assert file =~ """
+                 Erlang 23.1.1+ and Elixir 1.11+
```

Comment:
Not sure to add these checks as it will break on each update

Reply:
yes, it will fail when we `upgrade the Erlang version or Elixir version` for the template but that is what we want to have thus we would like to make sure the README is up to date along with the new version too, so we need to adjust the version in this test too :)